scripts/dotnet-cli-build/PublishTargets.cs

```diff
@@ -24,6 +26,8 @@ public static class PublishTargets
 
         private static string SharedFrameworkNugetVersion { get; set; }
 
+        private static List<string> _latestBlobs;
```

Comment:
No, no reason to re-open.

Looking again, it should be "s_" prefixed to follow the coding style here https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/coding-style.md